blogs/cli/index.md

```diff
@@ -0,0 +1,444 @@
+serviceaccount/netobserv-cli created
+clusterrole.rbac.authorization.k8s.io/netobserv-cli configured
+clusterrolebinding.rbac.authorization.k8s.io/netobserv-cli unchanged
+creating collector service
+service/collector created
+creating flow-capture agents:
+env: pktdrop_enable, env_value: true
+env: dns_enable, env_value: true
+env: rtt_enable, env_value: true
+env: filter_enable, env_value: true
+env: filter_peer_ip, env_value: 10.129.0.48
+daemonset.apps/netobserv-cli created
+Waiting for daemon set "netobserv-cli" rollout to finish: 0 of 2 updated pods are available...
+Waiting for daemon set "netobserv-cli" rollout to finish: 1 of 2 updated pods are available...
+daemon set "netobserv-cli" successfully rolled out
+Running network-observability-cli get-flows... 
+pod/collector created
+pod/collector condition met
+```
+
+Once that done, it will connect to the collector and display its output:
+```
+Starting network-observability-cli:
+=====
+Build Version: 
+Build Date: 
+
+INFO[0000] Running network-observability-cli
+Log level: info
+Filter(s): --enable_pktdrop=true --enable_dns=true --enable_rtt=true --enable_filter=true --peer_ip=10.129.0.48 
+INFO[0000] Starting Flow Capture...                     
+INFO[0000] Creating database...                         
+INFO[0000] flows.db created                             
+INFO[0000] Create flows table...                        
+INFO[0000] flows table created
+```
+
+At this stage, the collector wait for incoming data. If nothing shows yet, it means that no traffic is captured. Try to open the route of your application or update the filters of the capture.
+
+Once some traffic is captured, the output will look like:
+![cli dropped](./images/connectivity_scenario_cli_dropped.png)
+
+You can cycle to different views using left / right arrow keys and change the displayed enrichment colomns using page up / down ones.
+Also, to adapt to your screen height, you can increase / decrease the number of displayed flows using up / down arrow keys.
+
+In this capture, we see that the traffic is blocked by OVS since it reports the `OVS_DROP_LAST_ACTION` drop cause. This probably means that a Network Policy is involved.
+Edit your network policies and give another try.
+
+Behind the scenes in our scenario, we used to have a deny all on the pod label:
+```
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: deny-nodejs
+  namespace: connectivity-scenario
+spec:
+  podSelector:
+    matchLabels:
+      app: nodejs-basic
+  policyTypes:
+    - Ingress
+    - Egress
+```
+
+Once you updated your policies, you can give another try to your route until you fix the issue:
+![cli ok](./images/connectivity_scenario_cli_ok.png)
+
+The drop cause will dissapear and your route should open correctly now. On top of that, you can ensure that the Round Trip Time is correct. 
+If you are still experienting issues with the route, you may update / get rid of the filter(s) and play with live filtering.
```

Comment:
just a suggestion, here and in other places where you use "(s)"
```suggestion
If you are still experienting issues with the route, you may update / get rid of the filters and play with live filtering.
```

note, I _think_ it's fine in English to use plural when either singular or plural would work, ie. writing "filters" instead of "filter(s)", which is less cumbersome. cc @skrthomas , is it correct?